mentors/src/data_pipeline/pipeline_entrypoint.py

```diff
@@ -29,25 +28,25 @@ def process_session_data(args):
     mentor_dir: directory containing raw data for mentor
     """
     process_summary = {"transcripts": [], "audiochunks": []}
-    mentor_dir = MENTOR_DIR.format(args.mentor)
+    mentor_dir = os.path.join(os.getcwd(), MENTOR_DATA.format(args.mentor))
```

Comment:
probably better not to assume what `cwd` is in python scripts. Maybe we can put it in a variable and then reuse as needed?

If you're expecting it to be the `<project_root>/mentor` directory, could be like

```
MENTOR_DIR = os.environ.get("MENTOR_DIR") or "/app/mentor"
```